pkg/make/builder.go

```diff
@@ -582,6 +588,10 @@ func (b *Builder) BuildCmd(src, appName string) func(context.Context) error {
 			return err
 		}
 
+		if stderr.Len() > 0 {
+			return errors.New("stderr not empty")
```

Comment:
should we add the contents of stderr to the error?

Reply:
I decided not too -- the STDERR might be hundreds of lines. And since it's written to `os.Stderr` I don't think it's adds much here